docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
+template: concept-topic-template
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
+* Abstract products
+ 
+  * [Retrieving abstract products](/docs/scos/dev/glue-api-guides/202204.0/managing-products/abstract-products/retrieving-abstract-products.html)
+  
+  * [Retrieving image sets of abstract products](/docs/scos/dev/glue-api-guides/202204.0/managing-products/abstract-products/retrieving-image-sets-of-abstract-products.html)
+  
+* Concrete products
```

Comment:
```suggestion
* Concrete products:
```

Reply:
If your idea about these lists does not need a colon, don't apply this :)

Reply:
I added the colons : )